ts/store/actions/messages.ts

```diff
@@ -139,6 +140,16 @@ export const migrateToPaginatedMessages = createAsyncAction(
   "MESSAGES_MIGRATE_TO_PAGINATED_FAILURE"
 )<MessagesStatus, number, MigrationResult>();
 
+export const loadThirdPartyMessagePrecondition = createAsyncAction(
```

Comment:
Maybe "_load_" is not the right term for the action-triggered behaviour? After the `.request` is called, we trigger a web request so maybe a _getThirdPartyMessagePrecondition_ is better suited?